### ISSUE TITLE
Mark parenthetical (and other excludable) passages in dictionary entries

### ISSUE THREAD

**Issue by aarppe** [UAlbertaALTLab/cree-intelligent-dictionary#705]
So that we might exclude them for phrase generation (#704), parenthetical passages need to be explicitly marked so in the dictionary source definitions.

**Comment by aarppe** [comment]
@dwhieb A complete frequency list of parenthetical expressions for CW can be extracted with the following command:

`cat ~/altlab2/crk/dicts/Wolvengrey.tsv| cut -f4 | gawk '{ match($0,"[\\[\\(][^\\[\\(]+[\\]\\)]",f); if(f[0]!="") print f[0]; }' | sort | uniq -c | sort -nr > ~/altlab2/crk/generated/CW_parentheticals.txt`

The file is attached. [CW_parentheticals.txt](https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/files/6160236/CW_parentheticals.txt)

A list of partial/full parentheticals can be extracted from MD as follows:

`cat ~/altlab2/crk/dicts/Maskwacis.tsv | cut -f4 | gawk '{ if(match($0,"\\(?\\<(Also|Or|And|Animate|Inanimate|verb|noun).+$",f)!=0) print f[0]; }' > ~/altlab2/crk/generated/MD_parentheticals.txt`

The resultant file is attached here:  
[MD_parentheticals.txt](https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/files/6160274/MD_parentheticals.txt)

Parenthesized expressions in CW that should be included are (mainly prepositions or their abbreviations):
- [ ] (it/him)
- [ ] (it)
- [ ] (s.o. as)
- [ ] (s.t.)
- [ ] (it as)
- [ ] (them)

**Comment by dwhieb** [comment]
@aarppe That's super helpful - thanks!

Some (perhaps all) of this might get handled on the dictionary database side of things, in which case I'll migrate this issue to that repo.

**Comment by aarppe** [comment]
@dwhieb Yes, the parenthetical should primarily be coded in the dictionary database (ideally automatically while importing, with minimal but probably still necessary manual follow-up).

**Comment by dwhieb** [comment]
The new import JSON that I generated last week has extracted the parentheticals into a separate field. Once that data file is used in production we can mark this issue as closed.

Future work will be to add the parentheticals back in to the display in itwêwina. That's trivial for me to do on the database side, but will require changes to the morphodict code to be able to handle both "definition" and "notes" fields (which could be relatively easy or complicated depending on how we do it).
